app/views/menus/index.html.erb

```diff
@@ -1,5 +1,12 @@
 <div class="h-screen flex">
   <side-navbar active-element="menus"></side-navbar>
-  <div class="container ml-64 px-20 py-10 overflow-hidden">
+  <div class="container ml-64 px-20 py-10 overflow-hidden bg-gray-100">
+      <menus-container :menus="<%= [
+          {"name": "Cumpleaños niños","totalPrice": 18000, "recipesQuantity": 2, "recipes": ["Pie", "Torta", "Té"]}.to_json,
```

Comment:
estos objetos están hardcodeados por ahora o van a ser siempre así? Lo pregunto porque si va a crecer esta lista y va a seguir estando hardcodeada en el código quizás conviene moverla a otro lado.

Si es por ahora y después va a ser info que sale de la base de datos entonces da lo mismo, aunque igual podrías por ejemplo usar el to json en un map en vez de en cada linea jaja

Reply:
Está harcodeado por mientras!! Cuando lo conectemos con el backend todo eso va a cambiar ya que la información se va a obtener mediante un get